### PR TITLE
Update the PSA porting guide:

### PR DESCRIPTION
**Pull request by LDong-Arm**
* Mention CMake/Mbed CLI 2 support
* Add general improvements to use of mbed-os-tf-m-regression-tests

### CODE REVIEW

File: docs/porting/security/psa_targets_in_mbed-os.md [hunk 1]
```diff
@@ -295,8 +298,18 @@ To build TF-M and create an Mbed OS pull request:
...
-1. Create an Mbed OS pull request against [https://github.com/ARMmbed/mbed-os](https://github.com/ARMmbed/mbed-os).
+1. In the directory of your target, create or update `CMakeLists.txt` to define how CMake builds your target. Make sure it links `s_veneers.o` to your target and calls `mbed_post_build_tfm_sign_image()` which generates a ready-to-flash image containing the bootloader + signed TF-M secure binary + signed Mbed OS application.
+
+    You can take a look at `targets/TARGET_ARM_SSG/TARGET_MUSCA_S1/CMakeLists.txt` as an example.
+
+1. Switch back to the `mbed-os-tf-m-regression-tests` directory, and follow `README.md` to build and run all tests _manually_.
+
+    Ensure the test results are consistent with what you get from the vanilla TF-M tests (i.e. without Mbed OS integration). Some tests may be expected to fail.
+
+    Once you have the expected results, you can add them to `test/logs/<new target>/` to enable _automated_ testing. The logs should minimal and be based on regular expressions, see `test/logs/ARM_MUSCA_S1/` for example.
```
Comment (evedon) [inline docs/porting/security/psa_targets_in_mbed-os.md]:
```suggestion
    Once you have the expected results, you can add them to `test/logs/<new target>/` to enable _automated_ testing. The logs should be minimal and based on regular expressions, see `test/logs/ARM_MUSCA_S1/` for example.
```

**Review by evedon — COMMENTED** [review]
Minor comment

File: docs/porting/security/psa_targets_in_mbed-os.md (hunk 1 above)
Comment (LDong-Arm) [reply docs/porting/security/psa_targets_in_mbed-os.md]: Good catch, done.

**Review by evedon — APPROVED** [review]
Thanks Lingkai

**Comment by evedon** [comment]
@donatieng could you merge this PR?

**Pull request merged**